labs/src/main/scala/org/scalalabs/basic/lab01/OOExercise.scala

```diff
@@ -40,6 +40,36 @@ import scala.language.implicitConversions
  *   of type [[org.scalalabs.basic.lab01.CurrencyConverter]]
  * - Use the implicit CurrencyConverter to do the conversion. 
  */
-class Euro {
 
-}
\ No newline at end of file
+class Euro(var euro:Int, var cents:Int = 0) extends Currency() {
+  def inCents:Int = euro + cents
+  
```

Comment:
The [Scala Style Guide](http://docs.scala-lang.org/style/types.html) recommends putting a space between the colon and the type name, like so:
```def inCents: Int = euro + cents```